spec/controllers/projects/compare_controller_spec.rb

```diff
@@ -23,6 +23,19 @@
     expect(assigns(:commits).length).to be >= 1
   end
 
+  it 'compare should show some diffs with ignore whitespace change option' do
+    get(:show,
+        namespace_id: project.namespace.to_param,
+        project_id: project.to_param,
+        from: ref_from,
+        to: ref_to,
+        w: 1)
+
+    expect(response).to be_success
```

Comment:
@stanhu could you tell me how to add a new branch(just whitespace diff) in compare_controller_spec.rb?